tests/components/hue/test_config_flow.py

```diff
@@ -130,7 +130,7 @@ def mock_constructor(host, websession, username=None):
     flow = config_flow.HueFlowHandler()
     flow.hass = hass
 
-    with patch('aiohue.discovery.discover_nupnp',
+    with patch('homeassistant.components.hue.config_flow.discover_nupnp',
```

Comment:
Should this be part of this PR?

Reply:
No. I just realized that we messed it up when we moved to moving deps out of functions. Will move it to another PR

Reply:
#23816